pkg/annotations/annotations.go

```diff
@@ -6,4 +6,8 @@ const (
 
 	// MultipleTemplatesSupportedAnnotation is an annotation that indicates whether multiple templates of the same kind are supported by the template's remediator
 	MultipleTemplatesSupportedAnnotation = "remediation.medik8s.io/multiple-templates-support"
+
+	// NodeNameAnnotation is an annotation that will be placed on the CRs of remediatiors who support multiple templates of the same remediator.
+	// This is done because the CR will no longer be created with the node name.
```

Comment:
nit: probably there is no need to explain why the annotation is needed, or we can say something like "This annotation
reports the name of the Node..."

Reply:
>probably there is no need to explain why the annotation

Hmm why not ?
Might not be mandatory by I think it's a NTH, especially since this annotation will not always be present.

Reply:
I was referring to 

> // This is done because the CR will no longer be created with the node name.

I was thinking that in a year from now (or further in the future) the change in the CR name won't be so significative anymore to have a reference in another package (it will just be _something we used to do_), but I don't have a strong opinion on that.

On the other side, _what the annotation will hold_ is more important. It is clear that it is a Node name :), but it could be further explained like "the name of the CR target Node", or something similar.

Reply:
I see your point, makes sense :+1: